packages/analytics/amplify_analytics_pinpoint_dart/lib/src/impl/analytics_client/endpoint_client/endpoint_client.dart

```diff
@@ -118,8 +184,6 @@ class EndpointClient {
     }
 
     _endpointBuilder.user = newUserBuilder;
-
-    await updateEndpoint();
```

Comment:
Why is this being removed?

Reply:
I don't think it's clear that calling setUser will also call updateEndpoint.  So to improve code clarity I removed it and updated the call here: 
```
 @override
  Future<void> identifyUser({
    required String userId,
    required AnalyticsUserProfile userProfile,
  }) async {
    _ensureConfigured();
    await _endpointClient.setUser(userId, userProfile);
    await _endpointClient.updateEndpoint();
  } 
```